index.Rmd

```diff
@@ -1,74 +1,545 @@
+                                             # HSIncome_t=c("B19001_001","B19001_002","B19001_003","B19001_004","B19001_005","B19001_006","B19001_007","B19001_008","B19001_009"),##how to do it in a quicker way??
+                                             medianHSincome_t="B19013_001",
+                                             medianHValue_t="B25077_001",
+                                             medianGRent_t="B25064_001",
+                                             Poverty_t="B17001_002",
+                                             PovertyTotal_t="B17001_001",
+                                             ForeignBorn_t="B05002_009",
+                                             NavitityTotal_t="B05002_001",
+                                             Profjobs_t="C24070_010",
+                                             jobsTotal_t="C24070_001"
+              ),year=year[i],geometry = TRUE,
+              cache_table=T)
+  # create a county column in tract dataset for the later merge with county data.
+  d1$CountyGEOID=36029
+  d1$Year=year[i]
+  tract.list[i]=list(d1)
+  
+  d2 = get_acs(geography = "county", state = "NY",
+               county = "Erie", variables = c(TotalP_c="B01001_001",
+                 TotalRace_c= "B02001_001",
+                                              White_c="B02001_002",
+                                              nonHispanicWhite_c="B03002_003",
+                                              nonHispanicWhiteTotal_c="B03002_001",
+                                              Homeowner_c="B25003_002",
+                                              renterHS_c="B25003_003",
+                                              TenureHSTotal_c="B25003_001",
+                                              collegedegree_c="B15003_022",
+                                              degreeTotal_c="B15003_001",
+                                              # HSIncome_c=c("B19001_001","B19001_002","B19001_003","B19001_004","B19001_005","B19001_006","B19001_007","B19001_008","B19001_009"),##how to do it in a quicker way??
+                                              medianHSincome_c="B19013_001",
+                                              medianHValue_c="B25077_001",
+                                              medianGRent_c="B25064_001",
+                                              Poverty_c="B17001_002",
+                                              PovertyTotal_c="B17001_001",
+                                              ForeignBorn_c="B05002_009",
+                                              NavitityTotal_c="B05002_001",
+                                              Profjobs_c="C24070_010",
+                                              jobsTotal_c="C24070_001"
+               ),year=year[i],geometry = FALSE,
+               cache_table=T)
+  
+  county.list[i]=list(d2)
+  
+  ###make the tract data into wide format from the long format
```

Comment:
This chunk is too long - if you have headers like this it's a good idea to break into a separate code chunk.  This will also help you figure out what's happening when there are errors.